src/js/controller/tracks-loader.ts

```diff
@@ -96,8 +110,11 @@ function xhrSuccess(xhr, track, successHandler, errorHandler) {
     }
 }
 
-function loadVttParser() {
-    return require.ensure(['parsers/captions/vttparser'], function (require) {
-        return require('parsers/captions/vttparser').default;
-    }, chunkLoadWarningHandler(301131), 'vttparser');
+function loadVttParser(): VTTParser {
```

Comment:
```suggestion
function loadVttParser(): Promise<VTTParser> {
```

Reply:
Woops, something funky happened in rebase. Fixing now

Reply:
Alright, added a type to allow this. We aren't really using `require.ensure` in the way specified by the the webpack documentation, so that may explain why `require.ensure` is marked as returning void in the webpack-env types package.